src/main/java/dateBases/dbUtils/ListCreator.java

```diff
@@ -0,0 +1,52 @@
+package dateBases.dbUtils;
+
+import dateBases.entitys.Developer;
+import dateBases.entitys.Project;
+import dateBases.enums.EntitysEnum;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ListCreator {
+
+    public static List iterateDevelResultSet(EntitysEnum entityName, ResultSet resultSet) {
```

Comment:
We live at the time of Java 12. So why didn't you use generics that were appeared in Java 5?
And this method looks too long. Please refactor it.

Reply:
; -) Good point!